src/mlpack/methods/ann/layer/dropout.hpp

```diff
@@ -47,108 +47,104 @@ namespace ann /** Artificial Neural Network. */ {
  * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
  *         arma::sp_mat or arma::cube).
  */
-template <
-    typename InputDataType = arma::mat,
-    typename OutputDataType = arma::mat
->
-class Dropout
-{
+template<
+                typename InputDataType = arma::mat,
+                typename OutputDataType = arma::mat
+        >
+class Dropout {
  public:
-  /**
```

Comment:
Looks like the indentation is wrong, take a look at https://github.com/mlpack/mlpack/wiki/DesignGuidelines for more information.

Reply:
Thanks for this reference!  So all the method declarations in public should be two spaces?

Reply:
Indentation are two spaces, realistically you can take a look at the existing classes for examples.